docs/UserGuide.md

```diff
@@ -305,61 +305,81 @@ List all the events stored in the database with their unique index number.
 
 Delete the task with the unique index of `00123`.
 
-### Edit a floating task: `edit`
+### Edit a floating task/event/dealine: `edit`
+
+* Edit command can only edit the parameters of each commands but cannot transform a task to another
+type. For example, `eidt` cannot transform a floating task to a event task.
+
+#### Edit an floating task:
 
 Edit a floating task to revise its name or priority.
 
-    edit "TASK_NAME" [n-NEW_NAME | p-PRIORITY]...
+    edit TASK_UNIQUE_INDEX [n-NEW_NAME | p-PRIORITY]...
 
-* Task name should be in a pair of quotation marks.
+* Quotation marks are not necessary for `NEW_NAME`.
 
 * `PRIORITY` should only be the interger ranges from `0` to `5`.
 
-* Here we assume that no duplicate task name is used among all the task.
-
 #### Examples
 
-* `edit "washing clothes" p-0`
+* `edit 00124 p-0`
```

Comment:
since during the tutorial, the tutor suggests a scenario where the task name is extremely long, and if there is one digit wrong the user has to retype which is even worse than doing an undo